benchmarks/utils/gen_metrics_json.py

```diff
@@ -0,0 +1,258 @@
+import argparse
+import csv
+import json
+import re
+
+# Ref valid unit:
+# https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html
+UNIT_MAP = {
+    "Count" : 'Count',
+    "Milliseconds" : 'Milliseconds',
+    "ms" : "Milliseconds",
+    "Megabytes" : 'Megabytes',
+    "MB" : 'Megabytes',
+    "Gigabytes" : 'Gigabytes',
+    "GB" : 'Gigabytes',
+    'Bytes' : 'Bytes',
+    "B" : 'Bytes',
+    "Percent" : 'Percent',
+    "s" : 'Seconds'
+}
+
+METRICS_NAME_SET = {
+    "GPUUtilization",
+    "GPUMemoryUtilization",
+    "GPUMemoryUsed",
+    "CPUUtilization",
+    "DiskAvailable",
+    "DiskUsed",
+    "DiskUtilization",
+    "MemoryUsed",
+    "MemoryUtilization",
+    "Requests2XX",
+    "Requests4XX",
+    "Requests5XX",
+    "PredictionTime",
+    "HandlerTime",
+}
+
+# Ref metrics json format
+# https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-metric-streams-formats-json.html
+def gen_metrics_from_csv(csv_dict, stats_file_path):
+    if csv_dict is None:
+        return
+
+    metrics_dict_list = []
+    for k, v in csv_dict.items():
+        if k == "TS throughput":
```

Comment:
There's a lot of code duplication here can we create an enum from

k values to MetricName and put it at hte top to avoid all the if conditions?